src/Apps/Sell/Routes/IntroRoute.tsx

```diff
@@ -1,70 +1,91 @@
 import * as React from "react"
-import { Box, Button, Flex, SkeletonBox, Spacer, Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Flex,
+  Join,
+  Separator,
+  Image,
+  Spacer,
+  Text,
+} from "@artsy/palette"
 import { useRouter } from "System/Hooks/useRouter"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { SubmissionHeader } from "Apps/Sell/Components/SubmissionHeader"
 
+export const introData = [
+  {
+    number: "1",
+    title: "Tell us about your work",
+    description:
+      "Start by adding an artist from our list of high demand artists. Include information such as year, medium, dimensions and materials.",
+    image: "https://files.artsy.net/images/warhol-banner.jpg",
```

Comment:
Mock images, will updated with real ones once they are uploaded.

Reply:
Are [these images](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw/Submission-%26-Pre-Consignment-Management?node-id=2036-30236&t=avZ4QmZc9hBVhR54-4) in Figma final or are we waiting for additional images from Barney? If they are final, I believe we should be able to export directly from Figma and upload via the same internal interface.

Reply:
Updated 🖼️